t/benchmarks/messages_test.go

```diff
@@ -51,31 +52,28 @@ func TestSendMessages(t *testing.T) {
 	batchSent := make(chan struct{})
 	envelopesSent := int64(0)
 	go func() {
-		for {
-			select {
-			case ev := <-envelopeEvents:
-				if ev.Event != whisper.EventEnvelopeSent {
-					continue
-				}
-
-				envelopesSent++
-
-				if envelopesSent%(*msgBatchSize) == 0 {
-					fmt.Printf("Sent a batch and %d messages\n", envelopesSent)
-					batchSent <- struct{}{}
-				}
-
-				if envelopesSent == *msgCount {
-					fmt.Println("Sent all messages")
-					close(batchSent)
-					return
-				}
+		for ev := range envelopeEvents {
+			if ev.Event != whisper.EventEnvelopeSent {
+				continue
+			}
+
+			envelopesSent++
+
+			if envelopesSent%(*msgBatchSize) == 0 {
```

Comment:
Had to look twice, sadly spaces around `%` aren't kept. So probably a little helping comment.